userspace/libsinsp/sinsp_filtercheck_fspath.cpp

```diff
@@ -349,7 +348,14 @@ uint8_t* sinsp_filter_check_fspath::extract(sinsp_evt* evt, OUT uint32_t* len, b
 			return NULL;
 		}
 
-		m_tstr = sinsp_utils::concatenate_paths(tinfo->get_cwd(), m_tstr);
+		if(m_tstr.front() != '/')
```

Comment:
once the problem explained below is fixed I believe we can safely revert this change.

Reply:
Follow up: I think we still need this as in this case we concatenate cwd and m_tstr, and else we still use the helper to possibly resolve the path.

Reply:
Can we use `std::filesystem::path::is_absolute` instead? Much more readable IMHO.
If i got this correctly, we are stating that if `m_tstr` is relative, we attach the current working directory, else we use it as is, right?

Reply:
> else we use it as is, right?

almost correct we still need to resolve the absolute path as absolute paths can still contain path traversal. Statement based on real data observed.